android/src/main/java/com/swmansion/reanimated/nativeProxy/EventHandler.java

```diff
@@ -20,12 +20,13 @@ private EventHandler(HybridData hybridData) {
   }
 
   @Override
-  public void receiveEvent(int targetTag, String eventName, @Nullable WritableMap event) {
+  public void receiveEvent(int emitterReactTag, String eventName, @Nullable WritableMap event) {
     String resolvedEventName = mCustomEventNamesResolver.resolveCustomEventName(eventName);
-    receiveEvent(targetTag + resolvedEventName, event);
+    receiveEvent(resolvedEventName, emitterReactTag, event);
   }
 
-  public native void receiveEvent(String eventKey, @Nullable WritableMap event);
+  public native void receiveEvent(
+      String eventKey, int emitterReactTag, @Nullable WritableMap event);
```

Comment:
I think we should get rid of all occurrences of `eventKey`.
```suggestion
      String eventName, int emitterReactTag, @Nullable WritableMap event);
```